### PR TITLE
tests(0.13_0.14) test ssl cert & snis migration

### PR DESCRIPTION
**Pull request by kikito**
```
./test.sh -b kong:0.12.1 -t kong-private:certs-2-db upgrade_paths/0.13_0.14
./test.sh -b kong:0.12.1 -d cassandra -t kong-private:certs-2-db upgrade_paths/0.13_0.14
```

### CODE REVIEW

File: upgrade_paths/0.13_0.14/before/001-ssl-certificates.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  [ "Create a ssl certificate with no snis attached",
+    [ "admin", "POST", "/certificates", {
+      "cert": "cert",
+      "key": "key"
+    } ],
+    [ 201 ]
+  ],
+
+  [ "cert_3",
```
Comment (bungle) [inline upgrade_paths/0.13_0.14/before/001-ssl-certificates.json:19]: Should this be `cert_2`?

File: upgrade_paths/0.13_0.14/before/001-ssl-certificates.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    [ "admin", "POST", "/certificates", {
+      "cert": "cert",
+      "key": "key"
+    } ],
+    [ 201 ]
+  ],
+
+  [ "Attach a server name to cert_2",
```
Comment (bungle) [inline upgrade_paths/0.13_0.14/before/001-ssl-certificates.json]: The above says `cert_3` and here it reads `cert_2`.

**Review by bungle — COMMENTED** [review]
Just some naming issues(?).

**Comment by hishamhm** [comment]
This is pretty old and refers to no-longer-supported versions. Closing it!

**Pull request closed without merging**